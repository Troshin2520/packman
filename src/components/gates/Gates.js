import React, {Component} from 'react';
import './Gates.less';

class Gates extends Component {

  constructor(props) {
    super(props);
  }

  render() {
    return (<div className="gates">
      <div></div>
    </div>);
  }
}

export default Gates;