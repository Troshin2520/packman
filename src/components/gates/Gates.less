.gates {
    width: 4rem;
    height: 4rem;
    background-color: lightgray;
    display: flex;
    justify-content: stretch;
    align-items: center;
}

.gates>div {
    width: 4rem;
    height: 0.5rem;
    background-color: black;
}
