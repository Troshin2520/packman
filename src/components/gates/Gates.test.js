import React from 'react';
import Renderer from 'react-test-renderer';
import Gates from './Gates';


describe('Gates tests', () => {

  it('Gates renders without crushing', () => {
    const component = Renderer.create(
      <Gates/>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});