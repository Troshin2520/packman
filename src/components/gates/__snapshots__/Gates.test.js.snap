// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gates tests Gates renders without crushing 1`] = `
<div
  className="gates"
>
  <div />
</div>
`;
